teachers_digital_platform/__tests__/search.test.js

```diff
@@ -5,8 +5,16 @@ const app = require( `${ BASE_JS_PATH }search.js` );
 
 const HTML_SNIPPET = `
 
-  <form id="search-form" action="." data-js-hook="behavior_submit-search">
-    <input id="searchText" type="text" autocomplete="off" class="a-text-input" name="q" placeholder="Enter your search term(s)" value="executive">
+  <form class="tdp-activity-search" id="search-form" action="." data-js-hook="behavior_submit-search">
+    <div class="input-contains-label">
+      <label for="searchText" class="input-contains-label_before input-contains-label_before__search">
+          <span class="u-visually-hidden">The term to search for</span>
+      </label>
+      <input id="searchText" type="text" autocomplete="off" class="a-text-input" name="q" placeholder="Enter your search term(s)" value="executive">
+      <label for="searchText" class="input-contains-label_after input-contains-label_after__clear u-hidden">
+          Clear<span class="u-visually-hidden">Close</span>
```

Comment:
You shouldn't need `<span class="u-visually-hidden">Close</span>` since that's just text inside the label for screenreaders (looks like that's wrong in our example too and should be "Clear"). Since you already have "Clear" in the label, the label has text.